src/playercards/PlayerCardPanelData.ttslua

```diff
@@ -108,7 +108,8 @@ INVESTIGATOR_GROUPS = {
   },
   ["Neutral"] = {
   	"Lola Hayes",
-  	"Charlie Kane"
+  	"Charlie Kane",
+		"Subject 5U-21"
```

Comment:
Misformatted

Reply:
Interesting, since I matched the existing indentation:
![image](https://github.com/argonui/SCED/assets/97286811/2244b605-e682-4b48-af99-0c7f36a11d10)
Copied the Charlie line to fix this